index.html

```diff
@@ -981,8 +981,19 @@ <h3>Concrete Lifecycle Example</h3>
 <a>verifiable credential</a>. Pat selects the alumni
 <a>verifiable credential</a>, which is then composed into a
 <a>verifiable presentation</a>. The <a>verifiable presentation</a> is sent to
-the <a>verifier</a> and <a>verified</a>.
-        </p>
+the <a>verifier</a> and <a>verified</a>. 
+</p>
+<p>Once verified as authentic and current, the seller of the season ticket
+   then validates that the issuer of the <a>verifiable credential</a> is 
+   recognized for the claim of alumni status&mdash;it is, as it is issued
+   by Example University&mdash;and that today's date lies within the
+   validity period defined by the values of the validFrom and validUntil
+   properties. Since the holder is expected to be the subject of the 
+   <a>verifiable credential</a>, the <a>verifier</a> also confirms that 
+   the id for the alumni claim matches the id of the creator of the 
```

Comment:
The actual check is the `id` of the `credentialSubject` is associated, in some way, with the creator of the verifiable presentation. The language here is a bit loose, but that might be ok. I don't have a concrete suggestion that would be easy to read here.

Reply:
"associated, in some way" is not correct.

This particular example is an example of redeeming an alumni discount from a university, in which case arbitrary association is not the business rule. Subject==Holder is.